### PR TITLE
apps/appgen: Add templates for a sample application of the task manager

### PR DESCRIPTION
**Pull request by tyha1115**
- Add templates for a sample application
Sample application using the task manager of TizenRT
- Modify the appgen.sh in order to select the type of sample application.
- Revise the README.md according to the added templates and procedures.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6a10ff9994f9a5ed852996587a5d09476025d1e3] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6a10ff9994f9a5ed852996587a5d09476025d1e3] - Code Rule Check (C++) OK.

File: apps/appgen/template_task_manager_sample_main.c_source [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ *
+ * Copyright ##YEAR## Samsung Electronics All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (tyha1115) [reply apps/appgen/template_task_manager_sample_main.c_source]: This `##YEAR##` is automatically changed into the current year of system.

File: apps/appgen/template_eventloop_sample_main.c_source [hunk 2]
```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ *
+ * Copyright ##YEAR## Samsung Electronics All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (tyha1115) [reply apps/appgen/template_eventloop_sample_main.c_source]: This `##YEAR##` is automatically changed into the current year of system.

**Comment by seinfra** [comment]
Target : [887567d85fa3a149492fcbc9373171cb94ca511d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [887567d85fa3a149492fcbc9373171cb94ca511d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8b06cc299e7b101c2e81a402f7e4c4e7bad78a68] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8b06cc299e7b101c2e81a402f7e4c4e7bad78a68] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@tyha1115 Please let me know the decision of eventloop template.

**Comment by seinfra** [comment]
Target : [90d86d289f90f987c549880da4d1dc06f34cc6bf] -

**Comment by seinfra** [comment]
Target : [90d86d289f90f987c549880da4d1dc06f34cc6bf] -

**Comment by seinfra** [comment]
Target : [787d45ce7f74dfbf9b9ca6122cf0eaeba9ab85d9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [787d45ce7f74dfbf9b9ca6122cf0eaeba9ab85d9] - Code Rule Check OK.

**Pull request merged**